diesel_dynamic_schema/src/column.rs

```diff
@@ -0,0 +1,55 @@
+use diesel::backend::Backend;
+use diesel::expression::{AppearsOnTable, Expression, NonAggregate, SelectableExpression};
+use diesel::prelude::*;
+use diesel::query_builder::*;
+use std::borrow::Borrow;
+use std::marker::PhantomData;
+
+#[derive(Debug, Clone, Copy)]
+/// A database table column.
+/// This type is created by the [`column`](struct.Table.html#method.column) function.
+pub struct Column<T, U, ST> {
+    table: T,
+    name: U,
+    _sql_type: PhantomData<ST>,
+}
+
+impl<T, U, ST> Column<T, U, ST> {
+    pub(crate) fn new(table: T, name: U) -> Self {
+        Self {
+            table,
+            name: name,
```

Comment:
This should remove the _clippy_ warning.
```suggestion
            name,
```